rdt-app/src/main/assets/json.form/rdt-capture-form.json

```diff
@@ -176,6 +176,13 @@
           "value": "true",
           "err": "Please select the RDT reader result"
         }
+      },
+      {
+        "key": "time_img_saved",
```

Comment:
Does this have any purpose? Why not use metadata `end`

Reply:
This is the exact time the image is saved.

This happens immediately it is captured, not when the form is saved.